app/views/demands/_create-by-support-demand-form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= simple_form_for(demand.dup) do %>
+  <input type="hidden" name="demand[demand_description]" value="<%= demand.demand_description %>" />
```

Comment:
🤔

Reply:
an alternative approach is to pass just the demand id to clone from and do the attribute cloning in some server-side method?